test-it/karma/fixtures/javascriptinvoke/CloseReference.via

```diff
@@ -0,0 +1,111 @@
+define(DynamicReferencesVI dv(VirtualInstrument (
+    Locals: c(
+        ce(dv(.ErrorCluster (true 100 'error' )) errorIn)
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) err)
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) getObjectReferenceError1)
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) getObjectReferenceError2)
+
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) noErrorInAndValidReferenceError)
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) errorInAndValidReferenceError)
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) noErrorInAndInvalidReferenceError)
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) errorInAndInvalidReferenceError)
+
+        e(.JavaScriptDynamicRefNum objectReference)
+        e(dv(.Boolean false) isNotAValidRefnum1)
```

Comment:
<nitpick> In general, this is good to follow. Double negation is typically harder to follow. Though the API is NotAValidRefnum, we could negate the output boolean and and print/assert booleans as isValidRefNum1. Or, even write a simple wrapper SubVI around IsNotANumPathRefnum that simply negates the output and returns and call the SubVI in all the places instead of IsNotANumPathRefnum

Reply:
Done. Thanks for the suggestion.